app/models/crate.js

```diff
@@ -10,6 +10,13 @@ export default class Crate extends Model {
   @attr recent_downloads;
   @attr('date') created_at;
   @attr('date') updated_at;
+  /**
+   * This is the default version that will be shown when visiting the crate
+   * details page. Note that this value can be `null`, which may be unexpected.
+   * @return {string}
```

Comment:
```suggestion
   * @type {string | null}
```

I think `@return` is only meant for functions, no?

Reply:
Indeed. I wasn't aware of this part.